### ISSUE TITLE
vector pr-str

### ISSUE THREAD

**Issue by FalseProtagonist** [basilisp-lang/basilisp#1093]
pr-str on long vectors gives ["blah" "blah" ...]
as in an explicit ... symbol
makes sense for str but for pr-str should be isomorphism with read-str

**Comment by ikappaki** [comment]
to reproduce

```clojure
```

**Comment by FalseProtagonist** [comment]
thanks for reproduction, it's because the runtime provides defaults for the value by looking the dynamic vars up in core ns

basilisp.lang.runtime/lrepr:
print_length=core_ns.find(  # type: ignore
            sym.symbol(PRINT_LENGTH_VAR_NAME)
        ).value,
        
can workaround with:
(require '[basilisp.core :refer [*print-length*]])
(alter-var-root #'*print-length* (constantly nil))

don't know how it works in java-clojure but my intuition is that pr-str should ignore print-length completely

**Comment by ikappaki** [comment]
The initial value for `*print-length*` in Clojure in `nil`

```clojure
> clj
Clojure 1.12.0
user=> *print-length*
nil
(binding [*print-length* 5] (pr-str (range 10)))
"(0 1 2 3 4 ...)"
```

while in Basilisp is set to 50
```clojure
> basilisp repl
basilisp.user=> *print-length*
50
```

PR to follow (if @chrisrink10 is not already looking at this)

**Comment by chrisrink10** [comment]
Go ahead @ikappaki

**Comment by chrisrink10** [comment]
Fixed in #1095
